### PR TITLE
replace all misnamed occurrences of AGL to AHL

### PR DESCRIPTION
**Pull request by vasarhelyi**
Since flockwave-gps 2.3.1, flockwave-spec 1.70.0 and skybrush-server 2.10.11 the previous misnamed treatment of altitudes is finally corrected and now coordinates are expressed either as AHL (above home level, usually measured by pressure sensor etc.) or AGL (above ground level, usually measured by proximity sensors etc.). Previous references to AHL altitudes were incorrectly named as AGL due to an early Flat Earth assumption throughout the system. This pull request replaces all misnamed occurrences of AGL and ground-reference to AHL and home-reference, along with the protocol changes of the above mentioned versions of the backend.

### CODE REVIEW

File: src/model/sorting.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ export const UAVSortKey = Object.freeze({
   BATTERY: 'battery',
   GPS_FIX: 'gpsFix',
   ALTITUDE_MSL: 'amsl',
-  ALTITUDE_GROUND: 'agl',
+  ALTITUDE_HOME: 'ahl',
```
Comment (ntamas) [inline src/model/sorting.js:18]: This renaming can be a problem if the user selected AGL as the sorting key in the UAVs list in a previous version of Skybrush Live, closed the app, upgraded to this version and re-opened the app (with the old, saved sort key). It needs to be tested whether it causes a crash upon next startup or not. (Maybe a task for @Isti115 ? :)).

Comment (vasarhelyi) [reply src/model/sorting.js:18]: Maybe we can add support also for a new true AGL column and then both keys can be present

Comment (ntamas) [reply src/model/sorting.js:18]: yes, that's a possible alternative solution

**Comment by vasarhelyi** [comment]
As soon as @Isti115 pushes the president font size for the altitude status widget :) I think we will be ready for review...

**Comment by Isti115** [comment]
@vasarhelyi Done.

**Review by ntamas — APPROVED** [review]

**Pull request merged**